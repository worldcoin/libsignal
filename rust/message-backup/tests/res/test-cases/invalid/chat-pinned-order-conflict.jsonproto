[
  {
    "version": "1",
    "backupTimeMs": "1705692409729"
  },
  {
    "recipient": {
      "id": "1",
      "contact": {
        "aci": "IiMfftM7TXC2Brinq8Wbvw==",
        "pni": "kgNIuOIxTxmzIYUp0E+4Og==",
        "e164": "18045550102",
        "registered": {},
        "profileKey": "a9y+KGrHvMM94LKU4dMlFVjFa36zyjxsCv1vcgvjsOs="
      }
    }
  },
  {
    "recipient": {
      "id": "2",
      "self": {}
    }
  },
  {
    "recipient": {
      "id": "3",
      "releaseNotes": {}
    }
  },
  {
    "chat": {
      "id": "1",
      "recipientId": "2",
      "pinnedOrder": "5"
    }
  },
  {
    "chat": {
      "id": "2",
      "recipientId": "3",
      "pinnedOrder": "5"
    }
  }
]
