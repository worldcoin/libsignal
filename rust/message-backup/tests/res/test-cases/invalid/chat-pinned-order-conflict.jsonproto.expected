chat frame ChatId(2) error: PinOrder(5) already appeared